### PR TITLE
Getter instead of conversion in timestamped_video_frame

### PR DESCRIPTION
**Pull request by DaddyWesker**
It works for me currently. But PR requires testing on Mac and also on a slower machines (since it was laggy previously as i remember). Also chat messages works better for me at least. Please confirm it works better on your machines too.

### CODE REVIEW

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 1]
```diff
@@ -82,4 +79,7 @@ def __init__(self, message: TimestampedUnsignedCharVector,
         self.calibrationMatrix = np.reshape(np.asarray(loadedjson['projectionMatrix'], dtype=np.dtype(numpy.float32)), (4,4))
         jo_len = jo_len + 4
         received_img_bytes = message.data[jo_len:]
-        self.pixels = cv2.cvtColor(cv2.imdecode(np.asarray(bytearray(received_img_bytes), dtype="uint8"), cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)
\ No newline at end of file
+        self._pixels = received_img_bytes
+
```
Comment (noskill) [inline tagilmo/VereyaPython/timestamped_video_frame.py:83]:
```suggestion
    @property
```

Comment (noskill) [reply tagilmo/VereyaPython/timestamped_video_frame.py:83]: making it a property old code can read pixels like before, like pixels is a field

Comment (noskill) [reply tagilmo/VereyaPython/timestamped_video_frame.py:83]: @vsbogd i am not 100% sure about this, maybe it is better to let know api user that he or she calls a function here, what do you think?

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 2]
```diff
@@ -82,4 +79,7 @@ def __init__(self, message: TimestampedUnsignedCharVector,
...
+    def pixels(self):
+        return cv2.cvtColor(cv2.imdecode(np.frombuffer(self._pixels, dtype="uint8"), cv2.IMREAD_COLOR), cv2.COLOR_BGR2RGB)
```
Comment (Necr0x0Der) [inline tagilmo/VereyaPython/timestamped_video_frame.py:85]: Is it necessary to convert colors? If I just `imshow` the results from `pixels()` the image colors are wrong with this conversion. If I remove this conversion, then everything is ok. If the conversion is not needed on Mac, but is needed on other platforms, then we need to make this conversion platform-dependent. Could you please check this on different platforms?

**Comment by Necr0x0Der** [comment]
Thanks! Works for me in general - chat commands are lost from time to time, but not as often as before. Images can be retrieved, although the old question about color conversion should be finally solved...

File: tagilmo/VereyaPython/timestamped_video_frame.py (hunk 1 above)
Comment (vsbogd) [reply tagilmo/VereyaPython/timestamped_video_frame.py:83]: To me it is a matter of taste. Properties are definitely convenient for replacing a field by a method. If you can replace the field by a call in a client code you can just replace it. If you cannot (for instance you have a published API and cannot fix all clients) then the `@property` is the only way.

File: tagilmo/VereyaPython/timestamped_video_frame.py (hunk 2 above)
Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py]: @Necr0x0Der  well, I've changed that code to be platform-dependent, though i have to say, that on virtual mac (sosumi) this cv2.cvtColor is needed and with current platform-dependent conversion it gives wrong images (bgr).

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py]: @Necr0x0Der well, it seems that we don't need this cvtcolor after all. I've re-checked it once more. On sosumi too. So i've just deleted this conversion.

File: tagilmo/VereyaPython/timestamped_video_frame.py [hunk 3]
```diff
@@ -10,8 +10,8 @@
 import numpy.typing as npt
 import cv2
 import io
-import PIL.Image as Image
 import os
+import platform
```
Comment (Necr0x0Der) [inline tagilmo/VereyaPython/timestamped_video_frame.py:14]: Do we still need `import platform` here if there is no platform-dependent code after all?

Comment (DaddyWesker) [reply tagilmo/VereyaPython/timestamped_video_frame.py]: @Necr0x0Der  I forgot it. Sorry. Now it's gone

**Review by noskill — APPROVED** [review]

**Pull request merged**